library/core/src/cell.rs

```diff
@@ -1532,7 +1532,7 @@ impl<T: ?Sized + fmt::Display> fmt::Display for RefMut<'_, T> {
 /// `UnsafeCell<T>` is a type that wraps some `T` and indicates unsafe interior operations on the
 /// wrapped type. Types with an `UnsafeCell<T>` field are considered to have an 'unsafe interior'.
 /// The `UnsafeCell<T>` type is the only legal way to obtain aliasable data that is considered
-/// mutable. In general, transmuting an `&T` type into an `&mut T` is considered undefined behavior.
```

Comment:
This depends how you pronounce `&T`. I say "a reference type", but it sounds like the original author pronounced it "and-t".

Git says the author was @alexcrichton (e5da6a71a6a0b46dd3630fc8326e6d5906a1fde6)
 - Alex, what do you think?

Reply:
That could be true...but couple of lines later you can see it being written the way i did it.
https://github.com/rust-lang/rust/blob/2b9ba461116f24cfdbee5f3f4cc71185a7578639/library/core/src/cell.rs#L1552

Reply:
And then below again, it uses `an`: https://github.com/rust-lang/rust/blob/2b9ba461116f24cfdbee5f3f4cc71185a7578639/library/core/src/cell.rs#L1581

I think either they should all be consistent (and probably this would need a tracking issue because there's so many places it's inconsistent now), or it's not worth changing. I don't think it makes sense to just change this one article.

Reply:
If you create an issue, I can go for it.

Reply:
Sounds good, but we should decide whether to use 'a' or 'an' first. @steveklabnik do you have an opinion?

Reply:
I indeed pronounce this "and-t" and "and-mute-t", but I do not have an opinion on whether it's right or not.

Reply:
Yeah it's tricky because it technically depends on how you pronounce it. I do what @alexcrichton does most of the time, though often "amp" and not "and," not that that changes the rules here. 

https://github.com/rust-lang/rfcs/blob/master/text/1574-more-api-documentation-conventions.md is the canonical reference for how we word things. It does not directly address this issue, though it does contain the phrase "an `&str`," probably because of "string slice" rather than `&T` where `T` = `str`."

words are hard.

Reply:
> I indeed pronounce this "and-t" and "and-mute-t", but I do not have an opinion on whether it's right or not.

I honestly couldn't imagine how you pronouncing this and it being correct.
Sorry but can you write out the an exact phrase? The only thing I have in my head is `an reference type` and `an mutable reference type` which are obviously wrong.